events/2023-google-summer-of-code/index.markdown

```diff
@@ -65,6 +66,18 @@ For general MoveIt GSoC questions you can contact [Mark Moll](mailto:mark@pickni
 * Preferred skills: MoveIt 2, ROS 2, Motion Planning, C++
 * Mentor: [Sebastian Jahr](mailto:sebastian.jahr@picknik.ai), [Jafar Uruç](mailto:jafar.uruc@gmail.com)
 
+### Improved Realtime Control with MoveIt Servo
+
+* moveit_servo is the realtime control package of MoveIt. It is used for teleoperation and reactive or contact applications such as opening doors or generally interacting with any physical object with uncertain position. One of the downsides of Servo is that it only offers a ROS interface. The interface could be improved so that other C++ applications have better access to Servo; it would also be nice to document how a user can set custom IK goals. Specifically:
+  * Implement a C++ interface so that other C++ applications can instantiate and interact with Servo.
+  * Implement callbacks for pausing. This could be useful in scenarios such as opening a door, e.g. "Rotate 90* about X."
+  * Document how to use custom IK goal functions.
```

Comment:
Are you envisioning doing this using `pick_ik`? If so, might be worth explicitly mentioning and linking to it.

And if I understand that correctly, the project scope would 100% include creating custom IK goal functions as well.